stylize/__main__.py

```diff
@@ -32,10 +33,21 @@ def current_git_commit():
 ## Yields all files that differ from the branching point with @diffbase or are
 # not tracked by git.
 def enumerate_changed_files(exclude=[], diffbase="origin/master"):
-    # find common ancestor between @diffbase and @current_commit
-    ancestor = subprocess.check_output(
-        ['git', 'merge-base', current_git_commit(), diffbase]).strip().decode(
-            'utf-8')
+    try:
+        # try to find common ancestor between @diffbase and @current_commit
+        ancestor = subprocess.check_output(
+            ['git', 'merge-base', current_git_commit(), diffbase]).strip(
+            ).decode('utf-8')
+    except subprocess.CalledProcessError:
+        # There was no common ancestor between @diffbase and @current_commit
+        ancestor = subprocess.check_output(
+            ['git', 'rev-list', '--max-parents=0', current_git_commit()
+             ]).strip().decode('utf-8')
+        print("[ERR] Your diffbase: '" + str(diffbase) +
```

Comment:
This is a good idea, the way I did it was kind of a hack.

I've just pushed a new commit that (I think) does what you want. Lets see if it passes the tests? (I'm not sure how to run them)